src/gui/shader.cpp

```diff
@@ -1,13 +1,16 @@
 #include "shader.hpp"
 
+#include "../logging.hpp"
```

Comment:
target_include_directories(FunGame PRIVATE "${PROJECT_BINARY_DIR}")

Shouldn't this allow including logging from the src dir?

Reply:
No

![image](https://user-images.githubusercontent.com/76668284/208224329-293d4c6b-bfb2-47ee-9d1e-d61fed81b87c.png)